overrides/default.nix

```diff
@@ -1086,6 +1111,22 @@ lib.composeManyExtensions [
         }
       );
 
+      llama-cpp-python = super.llama-cpp-python.overridePythonAttrs (
+        old: {
+          propagatedBuildInputs = (old.propagatedBuildInputs or [ ]) ++ [
+            self.setuptools
+            self.scikit-build
```

Comment:
Any Python library that's needed to build or install the library, but not to run it, should go in `build-systems.json`.